packages/webchat/src/main.tsx

```diff
@@ -34,17 +34,23 @@ class Web extends React.Component<MainProps> {
   }
 
   async componentDidMount() {
+    // TODO: 1 - we need to get this from somewhere else. CDN of audio file?
     this.audio = new Audio(`${window.ROOT_PATH}/assets/modules/channel-web/notification.mp3`)
+    // TODO: 2 - can we remove intl?
     this.props.store.setIntlProvider(this.props.intl!)
+
+    // TODO: 3 - we migth want to watch how much we use global variables with window. Why is this globally available
     window.store = this.props.store
 
+    // TODO: 4 - iframe api? Why do we need that?
     window.addEventListener('message', this.handleIframeApi)
     window.addEventListener('keydown', (e) => {
       if (!this.props.config!.closeOnEscape) {
         return
       }
       if (e.key === 'Escape') {
         this.props.hideChat!()
+        // TODO: 5 - what to do with emulator mode?
         if (this.props.config!.isEmulator) {
           window.parent.document!.getElementById('mainLayout')!.focus()
```

Comment:
This was added because keyboard shortcuts were broken when you used "esc", the focus was still on the webchat so no other shortcuts were working until you clicked somewhere else